src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -1,5 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
dir.props assumes the default configuration is "Debug" if one isn't set however that isn't true for a number of our projects. It used to work out because we would default OSGroup to Windows_NT if it was not set however we now set it to AnyOS by default so for projects that actually need their default to be Windows_NT need to set it before importing dir.props. This allows people to be able successfully build this project without additional parameters passed to the compiler when building. With the ".builds" files for the src projects this isn't as big of a deal but for all our existing test projects most of them need a default OS beyond AnyOS to actually compile and run successfully. 

Also the Configuration is passed to the compilation via VS if you build from VS.